includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -546,8 +546,12 @@ private function getLicenseUrl() {
 		return $licenseUrl;
 	}
 
+	private function getLocalHref( $hrefKey ) {
+		return DesignSystemSharedLinks::getInstance()->getLocalHref($hrefKey, $this->lang, $this->productInstanceId);
+	}
+
 	private function getHref( $hrefKey ) {
-		return DesignSystemSharedLinks::getInstance()->getHref( $hrefKey, $this->lang );
+		return DesignSystemSharedLinks::getInstance()->getHref($hrefKey, $this->lang);
```

Comment:
Minor: Same here, there should be spaces between the brackets per coding conventions, so can just undo this change.